modules/apps/27-interchain-accounts/host/keeper/keeper_test.go

```diff
@@ -88,6 +88,8 @@ func NewICAPath(chainA, chainB *ibctesting.TestChain, encoding string, ordering
 
 // SetupICAPath invokes the InterchainAccounts entrypoint and subsequent channel handshake handlers
 func SetupICAPath(path *ibctesting.Path, owner string) error {
+	path.EndpointA.IncrementNextChannelSequence()
```

Comment:
`SetupICAPath`  manually calls `chanOpenTry` but not `chanOpenInit`, so we force here the increase of the sequence number.